EssentialFeed/EssentialFeediOS/Feed UI/Controllers/FeedImageCellController.swift

```diff
@@ -11,11 +11,13 @@ public protocol FeedImageCellControllerDelegate {
 }
 
 public final class FeedImageCellController: FeedImageView {
+	private let onImageSelection: ()->Void
```

Comment:
The codebase convention is to use spaces between `->`. Please, update all occurrences such as:

```suggestion
	private let onImageSelection: () -> Void
```